wasm/index.html

```diff
@@ -32,6 +42,29 @@
     <h1>
       <img src="logo.png" alt="Gother World">
     </h1>
+    <p>
+      <a id="badge" href="#" onclick="shutdown()">STOP</a>
+      <a id="badge" href="#" onclick="handleKeyEvent('l', 76, 'keydown'); setTimeout(() => handleKeyEvent('l', 76, 'keyup'), 50);">LOAD</a>
+      <a id="badge" href="#" onclick="handleKeyEvent('s', 83, 'keydown'); setTimeout(() => handleKeyEvent('s', 83, 'keyup'), 50);">SAVE</a>
+
+      // Game Part
+      <a id="badge" href="?gamePart=0">0</a>
```

Comment:
Should badge be a class and not an id?

Reply:
no, class is evaluated before css and would not work while id is evaluated later. tbh - I'm not the css master, but what I know is class does not work ;)

Reply:
yeah.. used id because i know shit about css ;)

fixed btw